_data-prepper/pipelines/configuration/sinks/opensearch.md

```diff
@@ -50,45 +50,82 @@ pipeline:
-
-### Serverless options
+`region` | No | String | The AWS Region to use for credentials. Defaults to [standard SDK behavior to determine the Region](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/region-selection.html).
+`sts_role_arn` | No | String | The AWS Security Token Service (AWS STS) role to assume for requests to Amazon SQS and Amazon S3. Defaults to `null`, which will use the [standard SDK behavior for credentials](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/credentials.html).
+`sts_header_overrides` | No | Map | A map of header overrides that the IAM role assumes for the sink plugin.
+`sts_external_id` | No | String | The external ID to attach to AssumeRole requests from AWS STS.
+`serverless` | No | Boolean | **(Deprecated in Data Prepper 2.7. Use this option under the `aws` configuration instead.)** Determines whether the OpenSearch backend is Amazon OpenSearch Serverless. Set this value to `true` when the destination for the `opensearch` sink is an Amazon OpenSearch Serverless collection. Default is `false`.
+`serverless_options` | No | Object | **(Deprecated in Data Prepper 2.7. Use this option under the `aws` configuration instead.)** The network configuration options available when the backend of the `opensearch` sink is set to Amazon OpenSearch Serverless. For more information, see [Serverless options](#serverless-options).
+
+<!-- vale off -->
+## actions
+<!-- vale on -->
+
+The following options can be used inside the `actions` option.
+
+Option | Required | Type | Description
+:--- |:---| :--- | :---
+`type` | Yes | String | The type of bulk action to use if the `when` condition evaluates to true. Must be either `create`, `index`, `update`, `upsert`, or `delete`.
+`when` | No | String | A [Data Prepper expression]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/expression-syntax/) to conditionally evaluate whether an event will be sent to OpenSearch using the bulk action configured in `type`. When empty, the bulk action will be chosen automatically when the event is sent to OpenSearch.
+
+
+## Serverless options
 
 The following options can be used in the `serverless_options` object.
 
 Option | Required | Type | Description
 :--- | :--- | :---| :---
-network_policy_name | Yes | String | The name of the network policy to create.
-collection_name | Yes | String | The name of the Amazon OpenSearch Serverless collection to configure.
-vpce_id | Yes | String | The virtual private cloud (VPC) endpoint to which the source connects.
+`network_policy_name` | Yes | String | The name of the network policy to create.
+`collection_name` | Yes | String | The name of the Amazon OpenSearch Serverless collection to configure.
+`vpce_id` | Yes | String | The virtual private cloud (VPC) endpoint to which the source connects.
```

Comment:
If this is referring to a generic VPC, then it is correct as written. If it is referring to Amazon VPC, then it should be "The Amazon Virtual Private Cloud (Amazon VPC) endpoint".

Reply:
Generic. Theoretically, someone couple use AWS with their own non-AWS VPC.